apollo-router/Cargo.toml

```diff
@@ -236,7 +236,7 @@ tracing = "0.1.40"
 tracing-core = "0.1.32"
 tracing-futures = { version = "0.2.5", features = ["futures-03"] }
 tracing-subscriber = { version = "0.3.18", features = ["env-filter", "json"] }
-trust-dns-resolver = "0.23.2"
+hickory-resolver = "0.24.1"
```

Comment:
It makes me feel better if you order this in the correct alphabetic place... :)

Reply:
moved in groups with other packages starting with H